Package.swift

```diff
@@ -7,20 +7,25 @@
  */
 
 import PackageDescription
+import Foundation
+
+var products: [PackageDescription.Product] = [
+    .library(name: "Tracking", targets: ["CioTracking"]),
+    .library(name: "MessagingPushAPN", targets: ["CioMessagingPushAPN"]),
+    .library(name: "MessagingPushFCM", targets: ["CioMessagingPushFCM"]),
+    .library(name: "MessagingInApp", targets: ["CioMessagingInApp"])
+]
+
+if (ProcessInfo.processInfo.environment["CI"] != nil) {
+    products.append(.library(name: "InternalCommon", targets: ["CioInternalCommon"]))
+}
```

Comment:
I plan on writing a comment in this file to explain what this logic is about. 

A gist is: xcode requires including `InternalCommon` as a library in order to get test coverage data on the module's code. 

But we want to hide the `InternalCommon` from customers when they try to install our SDK into their app using SPM. Therefore, I am making this file dynamic so the `InternalCommon` module only shows up when the SDK is being installed on the CI server.

Reply:
Docs wrote.